html_css/Exercise_1/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Exercise_1</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body class="body_attributes">
+
+<div class="white_box">
+	<img class="logo" src="img/ex1_brand_icon.png">
+	<br>
```

Comment:
read about "block" and "inline" elements in HTML and learn how we can avoid these `<br>` tags